.devcontainer/setup-ex.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python2
```

Comment:
[Python2 has hit EOL](https://www.python.org/doc/sunset-python-2/). Let's switch to Python3

Reply:
I know... if I am not mistaken we don't have python3 in the base image.

Reply:
Yeah, that's because of Centos. @nikita-b, am I right?

Reply:
Yes, CentOS 7 has only Python2 by default. We can add it, but image will be bigger